backend/src/main/java/eu/bbmri_eric/negotiator/negotiation/NegotiationSpecification.java

```diff
@@ -65,7 +68,7 @@ private static Specification<Negotiation> initOrAnd(
    * @param states a List of NegotiationState to use as filter
    * @return a Specification to add as part of a query to filter Negotiations
    */
-  public static Specification<Negotiation> hasState(List<NegotiationState> states) {
+  public static Specification<Negotiation> hasState(List<NegotiationState> states, boolean not) {
```

Comment:
what is not? :D

Reply:
It is a flag used to negate the condition. I added it so I could do "hasState not DRAFT" instead of "hasState in [all other states]"

Reply:
ah, then please update the javadoc so it is clear what it does or come up with a different solution